Composer/packages/client/config/extensions.config.js

```diff
@@ -0,0 +1,103 @@
+const path = require('path');
+
+const PnpWebpackPlugin = require('pnp-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+
+const paths = require('./paths');
+
+module.exports = (webpackEnv) => {
+  const isEnvDevelopment = webpackEnv === 'development';
+  const isEnvProduction = webpackEnv === 'production';
+
+  return [
+    {
+      entry: {
+        'react-bundle': 'react',
+      },
+      mode: isEnvProduction ? 'production' : 'development',
+      // export react globally under a variable named React
+      output: {
+        path: path.resolve(__dirname, '../public'),
+        library: 'React',
+        libraryTarget: 'var',
+      },
+      resolve: {
+        extensions: ['.js'],
+      },
+      optimization: {
+        minimize: isEnvProduction,
+        minimizer: [
+          new TerserPlugin({
+            extractComments: false,
+          }),
+        ],
+      },
+    },
+    {
+      entry: {
+        'react-dom-bundle': 'react-dom',
+      },
+      mode: isEnvProduction ? 'production' : 'development',
+      // export react-dom globally under a variable named ReactDOM
+      output: {
+        path: path.resolve(__dirname, '../public'),
+        library: 'ReactDOM',
+        libraryTarget: 'var',
+      },
+      externals: {
+        // ReactDOM depends on React, but we need this to resolve to the globally-exposed React variable in react-bundle.js (created by webpack-react.config.js).
```

Comment:
update or remove the `(created by webpack-react.config.js)` or so that it doesn't confuse people 😄